core/PRP/Surface/plDynamicEnvMap.cpp

```diff
@@ -108,7 +108,7 @@ void plDynamicEnvMap::IPrcParse(const pfPrcTag* tag, plResManager* mgr) {
         fYon = tag->getParam("Yon", "0").toFloat();
         fFogStart = tag->getParam("FogStart", "0").toFloat();
         fRefreshRate = tag->getParam("RefreshRate", "0").toFloat();
-        fIncCharacters = tag->getParam("IncCharacters", "0").toUint();
+        fIncCharacters = tag->getParam("IncCharacters", "0").toBool();
```

Comment:
This will work, but might be clearer to have the default `"false"` instead of `"0"`.